docker/nginx-container/check_ssl.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+while true; do
+  sleep 3600
+  if [ "$(md5sum /etc/nginx/ssl/live/polkadot-explorer.atleta.network/fullchain.pem \
```

Comment:
Make these files paths as local constants.

Reply:
Sure. It will be more convenient this way. Thanks